src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/controller/AlunoController.java

```diff
@@ -14,17 +14,11 @@
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.AlunoDto;

```

Comment:
De acordo com a regra do Sonar java:S120, os nomes de pacotes não podem ser camelCase
```suggestion
import com.projetogerenciamentocurso.gerenciamentocurso.dto.AlunoDto;
```

Reply:
feito